spyder/app/tour.py

```diff
@@ -1162,8 +1231,35 @@ def set_tour(self, index, frames, spy_window):
 
         self.is_tour_set = True
 
+    def _handle_fullscreen(self):
+        if self.spy_window.isFullScreen() or self.spy_window.fullscreen_flag:
+              if sys.platform == 'darwin':
+                  self.spy_window.setUpdatesEnabled(True)
+                  msg_title = _("Request")
+                  msg = _("To run the tour, please press the green button on "
+                          "the left of the Spyder window's title bar to take "
+                          "it out of fullscreen mode.")
+                  QMessageBox.information(self, msg_title, msg,
+                                          QMessageBox.Ok)
+                  return True
+              if self.spy_window.fullscreen_flag:
+                  self.spy_window.toggle_fullscreen()
+              else:
+                  self.spy_window.setWindowState(
+                      self.spy_window.windowState()
+                      & (~ Qt.WindowFullScreen))

```

Comment:
```suggestion
            if sys.platform == 'darwin':
                self.spy_window.setUpdatesEnabled(True)
                msg_title = _("Request")
                msg = _("To run the tour, please press the green button on "
                        "the left of the Spyder window's title bar to take "
                        "it out of fullscreen mode.")
                QMessageBox.information(self, msg_title, msg,
                                        QMessageBox.Ok)
                return True
            if self.spy_window.fullscreen_flag:
                self.spy_window.toggle_fullscreen()
            else:
                self.spy_window.setWindowState(
                    self.spy_window.windowState()
                    & (~ Qt.WindowFullScreen))
```

There's an extra indent here. I suggest setting your editor to 4 space indents for Python files to avoid this problem and the extra work and bugs of manual indents.